util/util.go

```diff
@@ -227,7 +230,7 @@ func CreateMSWorkingDirectory(uid string) (*os.File, error) {
 	if err != nil {
 		log.Printf("Error creating metric sample export directory : %v", err)
 	}
-
+	//nolint gogas
```

Comment:
What is gogas? Is it gas? gosec?

Reply:
gosec .... typo.. fixed...

Reply:
In this case gosec it checking to see if that file is user-supplied and insufficiently validated / running in a context where it can read files that the user would not be able to otherwise. In this case the ed variable is created by the agent and acceptable to override gogas in this case.